app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryModel.kt

```diff
@@ -14,7 +14,8 @@ data class NewMedicalHistoryModel(
     val ongoingPatientEntry: OngoingNewPatientEntry?,
     val ongoingMedicalHistoryEntry: OngoingMedicalHistoryEntry,
     val currentFacility: Facility?,
-    val showDiagnosisRequiredError: Boolean
+    val showDiagnosisRequiredError: Boolean,
+    val buttonState: ButtonState?
```

Comment:
Could we rename this property to specific exactly which button's state is being tracked here?